lib/crypto.js

```diff
@@ -92,27 +92,27 @@ Hash.prototype._flush = function _flush(callback) {
   callback();
 };
 
-Hash.prototype.update = function update(data, encoding) {
-  encoding = encoding || exports.DEFAULT_ENCODING;
-  this._handle.update(data, encoding);
+Hash.prototype.update = function update(data, inputEncoding) {
+  inputEncoding = inputEncoding || exports.DEFAULT_ENCODING;
+  this._handle.update(data, inputEncoding);
   return this;
 };
 
 
-Hash.prototype.digest = function digest(outputEncoding) {
-  outputEncoding = outputEncoding || exports.DEFAULT_ENCODING;
+Hash.prototype.digest = function digest(encoding) {
+  encoding = encoding || exports.DEFAULT_ENCODING;
   // Explicit conversion for backward compatibility.
-  return this._handle.digest(`${outputEncoding}`);
+  return this._handle.digest(`${encoding}`);
```

Comment:
To sync with the docs - https://nodejs.org/api/crypto.html#crypto_hash_digest_encoding
https://github.com/nodejs/node/blob/2e7ccc252e6c76835e892d135a29c45eb91ab5a4/doc/api/crypto.md#L783
Changing the docs causes the `href`s to change